David-Lectamo/tarea-5/game.js

```diff
@@ -0,0 +1,235 @@
+const ordened = [
+  [1 ,2 ,3 ,4 ],
+  [5 ,6 ,7 ,8 ],
+  [9 ,10,11,12],
+  [13,14,15,  ],
+];
+
+const numbers = [
+  [13,4 ,15,7 ],
+  [14,1 ,9 ,11],
+  [6 ,8 ,10,3 ],
+  [5 ,2 ,12,  ],
+];
+
+function init(){
+  const cell00 = document.getElementById('cell00'); 
+  const cell01 = document.getElementById('cell01'); 
+  const cell02 = document.getElementById('cell02');
+  const cell03 = document.getElementById('cell03');
+  const cell10 = document.getElementById('cell10');
+  const cell11 = document.getElementById('cell11');
+  const cell12 = document.getElementById('cell12');
+  const cell13 = document.getElementById('cell13');
+  const cell20 = document.getElementById('cell20');
+  const cell21 = document.getElementById('cell21');
+  const cell22 = document.getElementById('cell22');
+  const cell23 = document.getElementById('cell23');
+  const cell30 = document.getElementById('cell30');
+  const cell31 = document.getElementById('cell31');
+  const cell32 = document.getElementById('cell32');
+  const cell33 = document.getElementById('cell33');
+
+  cell33.addEventListener('click', () => {
```

Comment:
@L3C74M0, then you should assign this events traversing the matrix otherwise you are repeating a lot of code